tests/local.test.js

```diff
@@ -16,14 +16,24 @@ describe('getPackageStats', () => {
     expect(result.size).toEqual(425)
     done()
   })
+  test('dependencySizes', async done => {
+    const result = await getPackageStats(
+      path.resolve('./fixtures/node_modules/resolve-test')
+    )
+    expect(result.dependencySizes.length).toEqual(2)
+    expect(result.dependencySizes[0].name).toEqual(
+      'resolve-test/nested-folder/another-nested-folder'
+    )
+    expect(result.dependencySizes[1].name).toEqual('resolve-test')
```

Comment:
Seems a bit funny that `resolve-test` is a dependency of itself, but I tested the behaviour with other packages and this seems to be consistent.